images/linux/scripts/installers/java-tools.sh

```diff
@@ -28,15 +31,28 @@ if isUbuntu20 ; then
     defaultLabel11="(default)"
 fi
 
-apt-get -y install zulu-8-azure-jdk=\*
-apt-get -y install zulu-11-azure-jdk=\*
-apt-get -y install zulu-12-azure-jdk=\*
-update-java-alternatives -s /usr/lib/jvm/zulu-8-azure-amd64
+apt-get -y install adoptopenjdk-8-hotspot=\*
+apt-get -y install adoptopenjdk-11-hotspot=\*
+apt-get -y install adoptopenjdk-12-hotspot=\*
+apt-get -y install adoptopenjdk-13-hotspot=\*
+apt-get -y install adoptopenjdk-14-hotspot=\*
+
+# Set Default Java version to 8.
+if isUbuntu16 || isUbuntu18 ; then
+update-java-alternatives -s /usr/lib/jvm/adoptopenjdk-8-hotspot-amd64
+fi
+
+# Set Default Java version to 11.
+if isUbuntu20 ; then
+update-java-alternatives -s /usr/lib/jvm/adoptopenjdk-11-hotspot-amd64
+fi
```

Comment:
update-java-alternatives -s /usr/lib/jvm/adoptopenjdk-${DEFAULT_JDK_VERSION}-hotspot-amd64

Reply:
Thank you, I have changed according to your comment.